multichannel_timeseries/multichan.ipynb

```diff
@@ -0,0 +1,1087 @@
+    "                                                 sizing_mode='stretch_both',\n",
+    "                                                 layout='fit_columns',\n",
+    "                                                 theme='default',\n",
+    "                                                 stylesheets=[\":host .tabulator {font-size: 12px;}\"],\n",
+    "                                                 text_align={'description': 'center'},\n",
+    "                                             )\n",
+    "                                            ),\n",
+    "                              max_width=325\n",
+    "                             )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above code creates a `WidgetBox` containing:\n",
+    "- Annotation controls (PanelWidgets) for adding, editing, or deleting annotations. Since we grouped the annotations, `PanelWidgets` will also include a 'Visible' widget to toggle visibility by group.\n",
+    "- An `AnnotatorTable` to display annotations in a tabular format.\n",
+    "\n",
+    "We also customize the table's appearance and functionality using `tabulator_kwargs`, adjusting pagination, sizing, layout, theme, style, and text alignment. Check our the [Panel Tabulator](https://panel.holoviz.org/reference/widgets/Tabulator.html) docs for more options."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Integrating Annotations with the Plots\n",
+    "\n",
+    "Finally, we'll overlay the annotations onto our existing multichannel timeseries plot and the minimap. This integration allows us to see the annotations directly on the plots and interact with them.\n",
+    "\n",
+    "Add new annotations by:\n",
+    "- Ensure that the '+' button is selected in the widgets.\n",
+    "- Ensure that 'Box select (x-axis)' is selected in the Bokeh Toolbar.\n",
+    "- Enter the name of the group that you want the annotation to be added to in the 'description' text box.\n",
+    "- Select a time range by clicking and dragging on the plot\n",
+    "- Edit or delete annotations using the provided widgets and table.\n",
+    "\n",
+    "Note, if you are using a persistent file on disk for the annotations, hit the triangle button to commit changes to the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "annotated_multichan = (annotator * curves_overlay_lttb).opts(responsive=True, min_height=700)\n",
+    "annotated_minimap = (annotator * minimap)\n",
+    "annotated_app = (annotated_multichan + annotated_minimap).cols(1).opts(shared_axes=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "<div class=\"admonition alert alert-info\">\n",
```

Comment:
I've re-added the interactivity warning on examples.holoviz.org. Up to you to see whether you also want this admonition, I'm fine either way!